### PR TITLE
chore: update pin entry screen on Buy and Swap flow

### PR DESCRIPTION
**Pull request by dangelovska**

### CODE REVIEW

File: brd-ios/breadwallet/src/ViewControllers/RootNavigationController.swift [hunk 1]
```diff
@@ -27,17 +27,11 @@ class RootNavigationController: UINavigationController, UINavigationControllerDe
         let tintColor: UIColor
         
         switch viewController {
-        case is AccountViewController,
-            is HomeScreenViewController,
-            is KYCCameraViewController,
```
Comment (kenan-mamedoff) [inline brd-ios/breadwallet/src/ViewControllers/RootNavigationController.swift:32]: Are we sure?

Comment (dangelovska) [reply brd-ios/breadwallet/src/ViewControllers/RootNavigationController.swift:32]: yes, it's the same color for default case - Background.two

**Review by kenan-mamedoff — APPROVED** [review]

File: brd-ios/breadwallet/src/ViewControllers/RootNavigationController.swift (hunk 1 above)
Comment (kenanmamedoff) [reply brd-ios/breadwallet/src/ViewControllers/RootNavigationController.swift:32]: @dangelovska 
image

Comment (kenan-mamedoff) [reply brd-ios/breadwallet/src/ViewControllers/RootNavigationController.swift:32]: image

Comment (kenan-mamedoff) [reply brd-ios/breadwallet/src/ViewControllers/RootNavigationController.swift:32]: IMG_2879

**Pull request merged**